client.lua

```diff
@@ -672,7 +672,6 @@ local function updateInventory(items, weight)
 	end
 
 	client.setPlayerData('inventory', PlayerData.inventory)
-	TriggerEvent('ox_inventory:updateInventory', changes)
```

Comment:
I am not 100% sure if this is left from testing `add` and `remove` options or if it's something with a similar purpose as `ox_inventory:currentWeapon`, for example.

Reply:
This is for other resources to listen for changes; can be used for add/remove functionality and metadata changes.